api/rest/v2/index.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+title: REST (v2)
+nav_order: 0
+parent: API
+has_children: true
+published: false
+permalink: /api/rest/v2
+---
+
+# Data Commons REST API
+
+The Data Commons REST API is a
+[REST](https://en.wikipedia.org/wiki/Representational_state_transfer) library
+that enables developers to programmatically access data in the Data Commons
+knowledge graph. This package allows users to explore the structure of the
+graph, integrate statistics from the graph into data analysis applications and
+much more.
+
+## Getting Started
+
+First time using the Data Commons API, or just need a refresher? Take a look at
+our [Getting Started Guide](/api/rest/v1/getting_started).
```

Comment:
could you add a todo here to point to a new version of this? just so we don't miss it.

Reply:
Done